### PR TITLE
Port over WarningPanel component

### PR DESCRIPTION
**Pull request by stefanalund**
## Hey, I just made a Pull Request!

Another well used component from Spotify's internal deployment.

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [x] Tests added for new functionality

### CODE REVIEW

**Review by OrkoHunter — DISMISSED** [review]
LGTM! 🚀 

Screen Shot 2020-04-15 at 14 57 50
Screen Shot 2020-04-15 at 14 57 43

File: packages/core/src/components/WarningPanel/WarningPanel.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  component: WarningPanel,
+};
+
+export const Default = () => (
+  <WarningPanel
+    title="Lineage Warning"
+    message={
+      <>
+        This entity is missing lineage data. If this is unexpected, please make
+        sure you have set up Lineage correctly by following{' '}
```
Comment (OrkoHunter) [inline packages/core/src/components/WarningPanel/WarningPanel.stories.tsx:32]: Question (feel free to ignore) : What is missing "lineage data"?

Is this just an example of any kind of warning? Perhaps the warning text can be simplified.

**Comment by stefanalund** [comment]
@OrkoHunter that's a good point. I just took an internal example, but I agree that it was a bit confusing. Made it more generic now!

**Review by OrkoHunter — APPROVED** [review]
Awesome @stefanalund ! ✨

**Review by Rugvip — APPROVED** [review]
... aaand make an issue that this should be TS :grin:

**Comment by stefanalund** [comment]
@Rugvip https://github.com/spotify/backstage/issues/554

**Pull request merged**